src/main/java/org/jetbrains/research/anticopypaster/utils/ComplexityMetrics.java

```diff
@@ -0,0 +1,49 @@
+package org.jetbrains.research.anticopypaster.utils;
+
+import org.jetbrains.research.extractMethod.metrics.features.FeaturesVector;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class ComplexityMetrics extends Flag{
+
+    public ComplexityMetrics(List<FeaturesVector> featuresVectorList){
+        super(featuresVectorList);
+        calculateAverageComplexityMetrics();
+    }
+
+    private float getComplexityMetricFromFV(FeaturesVector fv){
+        if(fv != null){
+            return fv.buildArray()[3];
+        } else {
+            return 0;
+        }
+    }
+
+    private void calculateAverageComplexityMetrics(){
+
```

Comment:
Nitpick but the extra newlines is a little weird and an easy fix. I'd also like to request some comments throughout this. Javadocs and maybe an inline for the switch statement would be lit.

Reply:
yeah I realized I left out comments after I opened the PR, I added those in with your other requests